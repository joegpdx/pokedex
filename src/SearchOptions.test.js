import React from 'react';
import renderer from 'react-test-renderer';
import SearchOption from './SearchOptions.js';

test('renders PokeItem.js correctly', () => {
  const tree = renderer
    .create(<SearchOption />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});