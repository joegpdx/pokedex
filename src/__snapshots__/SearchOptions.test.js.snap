// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SearchOption.js correctly 1`] = `
<div
  id="search"
>
  <form
    onSubmit={[Function]}
  >
    <input
      onChange={[Function]}
      value=""
    />
    <button>
      Search
    </button>
  </form>
</div>
`;
